src/components/swiper/index.js

```diff
@@ -56,6 +56,21 @@ const Swiper = ( props ) => {
 
 	const swiperWrapperRef = useRef( null );
 
+	/*
+		This helper function will determine if we are in the site-editor or the Gutenberg editor
+	*/
+	const determineDocumentObject = () => {
+		const siteEditorIframeContainer = document.querySelector( '.edit-site-visual-editor__editor-canvas' );
+
+		let currentDocument = document;
+
+		if ( siteEditorIframeContainer ) {
+			currentDocument = siteEditorIframeContainer.contentDocument;
+		}
+
+		return currentDocument;
```

Comment:
We should remove this variable and instead return the correct value. On line 68 we should return the `siteEditorIframeContainer.contentDocument` if truthy as expected here and on line 71 we should return `document`. This will avoid the extra variable and also it allows the variable on line 98 to be more contextually relevant.